Source/Fuse.Controls.Video/Video.uno

```diff
@@ -481,11 +481,18 @@ namespace Fuse.Controls
 			}
 		}
 
+		static Selector _positionName = "Position";
+		static Selector _durationName = "Duration";
 		static Selector _progressName = "Progress";
 
 		void OnProgressChanged(object sender, EventArgs args)
 		{
+			UpdateScriptClass(Duration);
+
+			OnPropertyChanged(_positionName);
+			OnPropertyChanged(_durationName);
```

Comment:
I realize it doesn't make much sense to notify a Duration change in `OnPropertyChanged`... will fix on monday